test/udp_backend_test.rb

```diff
@@ -148,11 +148,19 @@ def test_supports_key_value_syntax_on_statsite
     StatsD.key_value('fooy', 42)
   end
 
+  # For key_value metrics (only supported by statsite), the sample rate
+  # part of the datagram format is (ab)used to be set to a timestamp instead.
+  # Changing that to `sample_rate: timestamp` does not make sense, so we
+  # disable the rubocop rule for positional arguments for now,
+  # until we figure out how we want to handle this.
+
+  # rubocop:disable StatsD/PositionalArguments
   def test_supports_key_value_with_timestamp_on_statsite
     @backend.implementation = :statsite
     @backend.expects(:write_packet).with("fooy:42|kv|@123456\n")
     StatsD.key_value('fooy', 42, 123456)
```

Comment:
So all we really know about this timestamp parameter is from this test?

Reply:
Yeah - Statsite support was contributed at some point - we've never used it ourselves. So all we have are the tests. The statsite repo has a bit more info though.